tests/autoria/test_average.py

```diff
@@ -0,0 +1,51 @@
+import requests_mock
+import json
+
+from autoria.api import RiaAverageCarPrice
+
+
+class TestAverage:
+    """Tests for autoria average car price calculation."""
+
+    def test_average_minimal(self, ria_categories, ria_marks, ria_states,
```

Comment:
See below, there is a docstring `Car price can be calculated with minimal set of parameters.`.
So yes, this is the test to check how it works with mininal set of parameters.
More tests can be added to verify optional parameters.